interfaces/ICurve.sol

```diff
@@ -6,15 +6,21 @@ interface ICurvePool {
     // _use_underlying If True, withdraw underlying assets instead of aTokens
     function add_liquidity(uint[2] calldata amounts, uint min_mint_amount, bool _use_underlying) external;
     function add_liquidity(uint[2] calldata amounts, uint min_mint_amount) external;
+    function add_liquidity(uint[4] calldata amounts, uint min_mint_amount, bool _use_underlying) external;
+    function add_liquidity(uint[4] calldata amounts, uint min_mint_amount) external;
     function remove_liquidity_one_coin(uint _token_amount, int128 i, uint _min_amount) external returns (uint);
-    function remove_liquidity_one_coin(uint _token_amount, int128 i, uint _min_amount, bool _use_underlying) external returns (uint);
+    function remove_liquidity_one_coin(uint _token_amount, int128 i, uint _min_amount, bool _donate_dust) external;
```

Comment:
mmmm this really changes from _use_underlying to donate?

Reply:
Also done, back to the "original" interface =)